sdk/trace/provider.go

```diff
@@ -65,7 +65,7 @@ func NewProvider(opts ...ProviderOption) (*Provider, error) {
 		namedTracer: make(map[string]*tracer),
 	}
 	tp.config.Store(&Config{
-		DefaultSampler:       ProbabilitySampler(defaultSamplingProbability),
+		DefaultSampler:       AlwaysSample(),
```

Comment:
Could you also remove the `defaultSamplingProbability` constant from `sampling.go`? It seems to became unused with this change.

Reply:
Seems like you need to remove the constant to actually make the CI pass. :)